### PR TITLE
[SCRIPT][Znm] Claret

### PR DESCRIPTION
**Pull request by nesstea**
Here ya go!

### CODE REVIEW

File: scripts/zones/Mount_Zhayolm/mobs/Claret.lua [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+function onMobInitialize(mob)
+end;
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:addMod(MOD_REGEN, math.floor(mob:getMaxHP()*.004));
+    mob:addMod(MOD_BINDRES, 10);
```
Comment (bendangelo) [inline scripts/zones/Mount_Zhayolm/mobs/Claret.lua]: 10 isn't much that is only 10 magic evasion towards bind.

File: scripts/globals/mobskills/Fluid_Toss_Claret.lua [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	local dmgmod = 5.5;
+	local info = MobPhysicalMove(mob,target,skill,numhits,accmod,dmgmod,TP_DMG_VARIES,1,2,3);
+	local dmg = MobFinalAdjustments(info.dmg,mob,skill,target,MOBSKILL_PHYSICAL,MOBPARAM_SLASH,info.hitslanded);
+	target:delHP(dmg);
+    
+    -- Apply poison
+    if (target:hasStatusEffect(EFFECT_POISON) == false) then
+        target:addStatusEffect(EFFECT_POISON, 100, 3, math.random(3,6) * 3); -- Poison for 3-6 ticks.
+    else
+        if (target:hasStatusEffect(EFFECT_POISON) == true) then
```
Comment (bendangelo) [inline scripts/globals/mobskills/Fluid_Toss_Claret.lua]: Don't need this condition

File: scripts/globals/mobskills/Fluid_Toss_Claret.lua [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            if (target:getStatusEffect(EFFECT_POISON):getPower() < 100) then
+                target:addStatusEffect(EFFECT_POISON, 100, 3, math.random(3,6) * 3); -- Poison for 3-6 ticks.
```
Comment (bendangelo) [inline scripts/globals/mobskills/Fluid_Toss_Claret.lua]: You have to remove the poison first then overwrite. Poison will not overwrite itself

Comment (nesstea) [inline scripts/globals/mobskills/Fluid_Toss_Claret.lua]: You told me it did yesterday! :)

Comment (bendangelo) [inline scripts/globals/mobskills/Fluid_Toss_Claret.lua]: I just checked and poison doesn't overwrite itself. You sure this is retail? Mobs won't use skills this quick so by the time the next fluit toss hits the target probably won't have it. I say just do `MobPhysicalStatusEffectMove(mob, target, skill, effect, power, tick, duration)`.

You also need to check for resistances and shadow hits, the method above dose all that for you.

**Comment by bendangelo** [comment]
I like it. Sorry I screwed you over. I just formatted the entire mob skill list sql file. You'll have to copy and paste your changes.

File: sql/mob_skills.sql [hunk 4]
```diff
@@ -381,7 +381,7 @@ INSERT INTO `mob_skills` VALUES (332,107,'Death_Trap',1,30.0,2000,2000,4,0,0,0);
 -- INSERT INTO `mob_skills` VALUES (333,333,'Mortal_Ray',0,7.0,2000,1500,4,0,0,0);
 INSERT INTO `mob_skills` VALUES (334,334,'Goblin_Rush',0,6.0,2000,1500,4,0,0,1);
 INSERT INTO `mob_skills` VALUES (335,335,'Bomb_Toss',2,13.5,2000,2000,4,0,0,0);
-INSERT INTO `mob_skills` VALUES (336,336,'Bomb_Toss_Suicide',1,13.5,2000,2000,4,0,0,0);
+INSERT INTO `mob_skills` VALUES (336,336,'Bomb_Toss_Suicide',0,13.5,2000,2000,1,0,0,0);
```
Comment (bendangelo) [inline sql/mob_skills.sql]: This should be kept as 1 and 4. Don't know why this changed.

**Comment by xdemolish** [comment]
leave a comment when requested changes have been made

**Comment by bendangelo** [comment]
thanks looks good

**Pull request merged**